### PR TITLE
Expand beta banner. Show it once per use

### PR DESCRIPTION
**Pull request by cmc333333**
Also bumps jquery and compresses static images.

User initially (or when clicking the expand link) sees:
screen shot 2015-02-05 at 12 36 00 pm

After navigating to another page (or clicking the x) she sees:
screen shot 2015-02-05 at 12 36 14 pm

### CODE REVIEW

File: peacecorps/peacecorps/static/peacecorps/js/src/showOnce.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ **/
+'use strict';
+
+var $ = require('jquery');
+
+var showOnce = function($root, storage) {
+  $root.find('[data-show-once]').each(function() {
+    var $el = $(this),
+        key = 'show-once:' + $el.attr('data-show-once'),
+        seen = !!(storage && storage.getItem(key));
```
Comment (msecret) [inline peacecorps/peacecorps/static/peacecorps/js/src/showOnce.js:13]: What happens when storage is null?

File: peacecorps/peacecorps/static/peacecorps/js/src/tests/showOnce_test.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    t.end();
+  });
+  t.test('should be shown if keys not present', function(t) {
+    var $root = $('<div />'),
+        $el1 = $('<span data-show-once="firstId" />'),
+        $el2 = $('<span data-show-once="secondId" />'),
+        storage = mockStorage();
+    $root.append($el1, $el2);
+    showOnce.showOnce($root, storage);
+    console.log($el1);
```
Comment (msecret) [inline peacecorps/peacecorps/static/peacecorps/js/src/tests/showOnce_test.js]: Is there  a reason to logging in the test?

File: peacecorps/peacecorps/static/peacecorps/js/src/showOnce.js (hunk 1 above)
Comment (cmc333333) [inline peacecorps/peacecorps/static/peacecorps/js/src/showOnce.js:13]: Seen is false and it proceeds to show the beta warning. Originally it'd silently fail, but testing pointed out the issue.

File: peacecorps/peacecorps/static/peacecorps/js/src/tests/showOnce_test.js (hunk 2 above)
Comment (cmc333333) [inline peacecorps/peacecorps/static/peacecorps/js/src/tests/showOnce_test.js]: Good catch; just debugging.

**Comment by cmc333333** [comment]
Updated images for the generic header:
screen shot 2015-02-05 at 3 52 02 pm
screen shot 2015-02-05 at 3 52 11 pm

**Pull request merged**